README.md

```diff
@@ -48,6 +48,8 @@ DATABASE_USER=postgres
 DATABASE_PASSWORD=postgres
 
 SECRET=asdlplplfwfwefwekwself.2342.dawasdq
+
+MongoUri=mongodb://localhost:27017/mydatabase
```

Comment:
Does the *Testing* section from the README.md work as expected?

Reply:
![screen shot 2018-12-23 at 1 10 40 pm](https://user-images.githubusercontent.com/8157594/50383012-587d5700-06b4-11e9-8981-05303b7cf1e2.png)
the tests you wrote work but only for centain scenarios, the one ones that requires Id to be autoincremented id fail as mongodb uses mongoid. we would have to make the test deterministic

Reply:
Can we fix this for the MongoDB repository's tests with minimal changes? The PostgreSQL repository is okay for now.